spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1432,6 +1439,7 @@ bom {
 				"spring-boot-starter-logging",
 				"spring-boot-starter-mail",
 				"spring-boot-starter-mustache",
+				"spring-boot-starter-neo4j",
```

Comment:
I think this should be removed as there is no starter for the driver (and their shouldn't be any if all that's needed is the driver.